.drone.yml

```diff
@@ -0,0 +1,110 @@
+kind: pipeline
```

Comment:
Please run this through `drone fmt` to get some easier to read formatting on it.

Reply:
Updated